AlphaWallet/UI/Views/TokenCardRowView.swift

```diff
@@ -359,6 +362,12 @@ extension TokenCardRowView: TokenInstanceWebViewDelegate {
 	}
 
 	func reinject(tokenInstanceWebView: TokenInstanceWebView) {
-		//no-op
+		//Refresh if view, but not item-view
+        if isStandalone {
```

Comment:
opps :)

Reply:
You mean it was a bug? It's by design :P I thought it was supposed to only work for action views. We agreed on: "web3.action.setProps".

Reply:
Oh the tab+space. Damn relic.